connectivity/FEATURE_BLE/source/common/ble_trace_helpers.h

```diff
@@ -44,11 +44,38 @@ static inline const char* tr_as_array(T item)
     return (mbed_trace_array)((const uint8_t*)&item, sizeof(item));
 }
 
-static inline const char* to_string(UUID uuid)
-{
+static inline const char* to_string(const UUID &uuid) {
```

Comment:
why do you need both?

Reply:
Because it's called in Gatt Client and mbed-os won't compile otherwise.

Reply:
^ not applicable after commit 8f28da36be2c0da2b2abfb5ba2120a509d7bb5cc